appinventor/buildserver/src/com/google/appinventor/buildserver/BuildServer.java

```diff
@@ -652,6 +653,7 @@ public void run() {
               }
             } catch (Exception e) {
               // TODO(user): Maybe send a failure callback
+              e.printStackTrace();
```

Comment:
Combine with following line using LOG.log, which can take the exception for the purposes of printing the stack trace.